### PR TITLE
Maintainerr

### PR DESCRIPTION
**Pull request by maindust**
# Description

Maintainterr allows you to automate the recycling and clearing of HD space for those who might not want to expand or for those who want to expunge non watched content. See pages below. Very similar to Overseerr app.

https://docs.maintainerr.info/Introduction/

Created docs PR here. https://github.com/saltyorg/docs/pull/316

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.  You can use the checkboxes below or delete them as you wish.

- [ ] I've had this running in my dev and prod environments for 2 weeks a piece.
- [ ] Spun this up on 2 VM's and worked each time.
- [ ] Spun this up on VPS and ran for a week with no issues.

### CODE REVIEW

**Comment by owine** [comment]
Please cleanup the lint errors for CI

File: roles/maintainerr/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+# Commands
+maintainerr_docker_commands_default: []
+maintainerr_docker_commands_custom: []
+maintainerr_docker_commands: "{{ lookup('vars', maintainerr_name + '_docker_commands_default', default=maintainerr_docker_commands_default)
+                               + lookup('vars', maintainerr_name + '_docker_commands_custom', default=maintainerr_docker_commands_custom) }}"
+
+# Volumes
+maintainerr_docker_volumes_default:
+  - "{{ maintainerr_paths_location }}:/app/config"
```
Comment (powerdude) [inline roles/maintainerr/defaults/main.yml:116]: shouldn't the path be `/opt/data`?

Comment (maximuskowalski) [reply roles/maintainerr/defaults/main.yml:116]: Looks that way to me from the installation docs.

Comment (maindust) [reply roles/maintainerr/defaults/main.yml:116]: You're talking about here?

https://docs.maintainerr.info/Installation/#linux-permissions-example

**Pull request merged**